packages/monitoring/src/helpers/utils.ts

```diff
@@ -0,0 +1,37 @@
+import { RequestError } from "@threefold/types";
+import axios, { AxiosError } from "axios";
+
+import { TextColor } from "../types";
+
+export async function sendGetRequest(url: string, body: string, headers: Record<string, string>) {
```

Comment:
Get request doesn't have body ... also we don't need any headers here for now so we can mark headers as optional